PriscillaSam/Stackoverflow-Lite
# #160022206 Modify api endpoint to vote an answer

#### What does this PR do?
Modify api endpoint **POST** `api/v1/answers/:id` to interact with a database.

#### Description of Task to be completed?
- Connect endpoint to database.
- Write sql queries to add and update votes from database.
- Refactor tests to use database.

#### How should this be manually tested?
* Clone the repo using `git clone https://github.com/PriscillaSam/Stackoverflow-Lite.git`.
* run `git checkout ft-vote-answer-160022206 `.
* run `npm install` and `npm run start:dev`.
* Open up Postman and test **POST** `localhost:3000/api/v1/answers/:id`.
*Note*: You need to be authenticated to perform this operation. So signup by providing your name, email, password and confirm password or login using this sample data:
```gherkin
{
   email: "garry.doe@gmail.com",
   password: "password"
}
```

#### Any background context you want to provide?
None.

#### What are the relevant pivotal tracker stories?
#160022206
